build.sbt

```diff
@@ -34,7 +34,7 @@ lazy val scoverageSettings =
     ScoverageKeys.coverageHighlighting := true,
     coverageEnabled.in(ThisBuild, Test, test) := true
   )
-
+resolvers += "emueller-bintray" at "http://dl.bintray.com/emueller/maven"
```

Comment:
on line 60 there's a place where a resolver is already being added. You could do something like `resolvers +== Seq(...)` there so it's all in one place

Reply:
sorry for this one, fixed.